docs/api-restful.rst

```diff
@@ -33,16 +33,16 @@ Synopsis
 .. code-block:: bash
 
     # Download a main dataset JSON file…
-    curl https://nextstrain.org/seasonal-flu/h3n2/ha/2y                         \
+    curl https://nextstrain.org/zika                         \
```

Comment:
Whitespace to align `\` across multiple lines is a bit off now. I'll fix in another PR

Reply:
https://github.com/nextstrain/nextstrain.org/pull/943